Dockerfile

```diff
@@ -151,7 +155,12 @@ ENV GO_PATH=/usr/local/go/bin/go
 
 RUN curl -sL https://deb.nodesource.com/setup_20.x | bash - 
 RUN apt-get -y update && apt-get install -y curl nodejs awscli && apt-get clean \
-  && rm -rf /var/lib/apt/lists/*
+    && rm -rf /var/lib/apt/lists/*
+
+RUN apt-get update \
```

Comment:
The versions should be pinned when installing packages to the exact versions instead of upgrading otherwise it will mess with caching

Reply:
Yes, 

That's all the issue with docker images and security :(

Let's remove it for now. Just adding the possibility to remove git on demand is enough. (we wont use git sync)